tasks/section_3_Network_Configuration.yaml

```diff
@@ -530,7 +530,16 @@
         name: ufw
         state: started
         enabled: true
-      when: UFWEnable
+    
+    - name: 3.5.1.3 Ensure ufw service is enabled | If needed, reset any rules manually added to UFW before enable
```

Comment:
| Reset any rules previously added to UFW

Reply:
Ok